Toolchains/toolchain_x86_64.cmake

```diff
@@ -0,0 +1,19 @@
+# This is tuned for Ubuntu + gcc
+# Fun Fact: Ubuntu's i386 actually uses the i686 compiler
+
+set(CMAKE_SYSTEM_PROCESSOR x86_64)
+
+set(triple x86_64-linux-gnu)
+
+set(CMAKE_C_COMPILER x86_64-linux-gnu-gcc)
+set(CMAKE_CXX_COMPILER x86_64-linux-gnu-g++)
+
+set(CMAKE_C_COMPILER_TARGET ${triple})
+set(CMAKE_CXX_COMPILER_TARGET ${triple})
+
+set(CMAKE_FIND_ROOT_PATH /usr/x86_64-linux-gnu)
+
+set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
```

Comment:
Could you document what exactly we're setting these for?

Reply:
I just copied them from the cmake tutorial

Reply:
I'll look them up

Reply:
Added some rough explanations